Extension/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 0.17.8: August 16, 2018
 * Add `inactiveRegionForegroundColor` and `inactiveRegionBackgroundColor` to settings. [#2212](https://github.com/Microsoft/vscode-cpptools/issues/2212)
+* Add multi-pass environment variable resolution allowing variables defined in terms of other variables. [#2322](https://github.com/Microsoft/vscode-cpptools/pull/2322)
```

Comment:
We usually just try to do one line per github issue.  I would just keep this first line and remove the rest.